BsToMuMuGammaNTuplizer/plugins/BMMGUtil.h

```diff
@@ -0,0 +1,819 @@
+#ifndef __BMMGUtil_H__
+#define __BMMGUtil_H__
+
```

Comment:
Add the referance to Dimas repo
```suggestion
//  Many of these function are obtained from the Bmm groups ntuplizer codebase
//  see here https://github.com/drkovalskyi/Bmm5
```